src/github.com/getlantern/golog/golog_test.go

```diff
@@ -6,26 +6,34 @@ import (
 	"io/ioutil"
 	"os"
 	"regexp"
+	"strings"
 	"testing"
 	"time"
 
-	"github.com/getlantern/testify/assert"
```

Comment:
Looks like we initially forked this one in order to add a small change https://github.com/stretchr/testify/compare/master...getlantern:master.

The change was just adding more debugging to the output of a test, so it's OK to use the upstream package. We should probable deprecate `github.com/getlantern/testify` too if we don't plan to make major changes.